docs/nGraph_DG/nGraph_Python_API.md

```diff
@@ -0,0 +1,222 @@
+# Using nGraph's Python API
+
+nGraph is OpenVINO's graph manipulation library, used to represent neural network models in the form of a computational graph. Using nGraph's Python APIs, you can create, inspect and manipulate computational graphs. You can also use the API to set runtime properties of the graph, which will be used by OpenVINO during inference.
```

Comment:
nGraph is the OpenVINO™ graph manipulation library that is used to represent neural network models in the form of a computational graph. With nGraph Python APIs, you can create, inspect, and manipulate computational graphs. ...